src/main/kotlin/com/exactpro/th2/infraoperator/configuration/fields/RabbitMQManagementConfig.kt

```diff
@@ -25,6 +25,6 @@ data class RabbitMQManagementConfig(
     val username: String = "",
     val password: String = "",
     val persistence: Boolean = false,
-    val rubbishCollectionInterval: Int = 900,
+    val cleanUpOnStart: Boolean = true,
```

Comment:
question: Do we need this parameter at all? What is the case when we don't want to execute cleanup (remove everything that is not needed) on start?

Reply:
I thing we can remove it.